sdks/python/build.gradle

```diff
@@ -267,6 +273,7 @@ task portableWordCount(dependsOn: 'installGcpTest') {
             "--input=/etc/profile",
             "--output=/tmp/py-wordcount-direct",
             "--runner=PortableRunner",
+            "--experiments=worker_threads=100",
```

Comment:
by default we have 12 threads and all those were blocked in input at some time suggesting that it needs more threads.

Reply:
That default seemed to serve well, 100 is an arbitrary change. Will I need 1000 to process a large file? Have you analyzed the root cause? Is there a JIRA for it? Isn't it specific to batch? If so, then perhaps the GRPC threads get blocked due to how the responses are processed in the runner? We should undo this change and address the root cause. Otherwise this may just mask other issues and defeat the purpose of having the pre-commit signal.

Reply:
I think the local execution environment sets the parallelism of the pipeline same as the number of coders on the machine. In my case, its 12. As we have 12 threads in SDKHarness, we get a deadlock on my machine.
Running on a cluster does not have this issue as the default parallelism there is 1.

Reply:
In any case I think we should resolve of scheduling miss match between SDKHarness and Flink.

Reply:
So it can be reproduced by setting parallelism > 12 in batch mode? What scheduling mismatch do you refer to?

Reply:
I tested this locally. If I set parallelism to 16, the run gets stuck. With parallelism of 8, it passes. This applies only to batch.

Reply:
Reference: https://lists.apache.org/thread.html/0cbf73d696e0d3a5bb8e93618ac9d6bb81daecf2c9c8e11ee220c8ae@%3Cdev.beam.apache.org%3E

This is the same issue that we had with TFX.
Flink can run unbounded number of tasks on a JVM in parallel while Python SDK can only run 12 by default. This causes the issues where Flink submits a ProcessBundleRequest to SDKHarness while a downsteam ProcessBundleRequest has already taken up the tread.